File: CS/ViewModel/MainViewModel.cs
using DevExpress.Mvvm;
using System.Windows.Input;

namespace Example.ViewModel {
    public class MainViewModel {
        protected virtual IMessageBoxService MessageBoxService { get { return null; } }
        public void ShowMessage() {
            MessageBoxService.Show("This is MainView!");
        }
    }
}

File: VB/ViewModel/MainViewModel.vb
Imports DevExpress.Mvvm

Namespace Example.ViewModel

    Public Class MainViewModel

        Protected Overridable ReadOnly Property MessageBoxService As IMessageBoxService
            Get
                Return Nothing
            End Get
        End Property

        Public Sub ShowMessage()
            MessageBoxService.Show("This is MainView!")
        End Sub
    End Class
End Namespace
